app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -610,9 +610,10 @@ class Settings(private val appContext: Context) : PreferencesHolder {
     /**
      * Indicates if the total cookie protection CRF should be shown.
      */
-    var shouldShowTotalCookieProtectionCFR by booleanPreference(
+    var shouldShowTotalCookieProtectionCFR by lazyFeatureFlagPreference(
         appContext.getPreferenceKey(R.string.pref_key_should_show_total_cookie_protection_popup),
-        default = mr2022Sections[Mr2022Section.TCP_CFR] == true,
+        featureFlag = true,
```

Comment:
Not pretty but using a `lazyFeatureFlagPreference` means a quick (will have to be uplifted) and safe (used by the other MR features) general approach.